Sources/apm-agent-ios/AgentConfigBuilder.swift

```diff
@@ -104,6 +111,7 @@ public class AgentConfigBuilder {
     config.connectionType = connectionType
     config.managementUrl = self.managementUrl
     config.enableRemoteManagement = enableRemoteManagement
+    config.collectorPathPrefix = collectorPathPrefix
 
```

Comment:
I think it would be better to use the url component parser below to capture a path in url set in the export url, rather than adding a new builder method.